src/cmd/ksh93/ksh.1

```diff
@@ -7045,10 +7045,6 @@ is used, the variable
 is not unset first.
 .TP 8
 .B \-B
-Enable brace pattern field generation.
-This is the default behavior.
-.TP 8
-.B \-B
 Enable brace group expansion.  On by default.
```

Comment:
FWIW `set --help` has `Enable {...} group expansion. On by default.`. Which is why we need a single source of truth for such things.